Composer/packages/client/src/components/Page.tsx

```diff
@@ -101,6 +101,7 @@ type IPageProps = {
   projectId?: string;
   skillId?: string;
   dialogId?: string;
+  fileId?: string;
```

Comment:
I debated if it shouldn't just be fileId all the way down rather than having luFileId and lgFileId.  What do you think?

Reply:
it's already overloaded by props, kept it at minimum, it's called "Page" (reusable) so shouldn't have any knowledge about projectId, skillId, dialogId, lg/lu ids imho.